src/sharepoint/rest/lists.ts

```diff
@@ -4,16 +4,15 @@ import { Items } from "./items";
 import { Views, View } from "./views";
 import { ContentTypes } from "./contenttypes";
 import { Fields } from "./fields";
-import { Queryable } from "./queryable";
+import { Queryable, QueryableCollection, QueryableSecurable } from "./queryable";
 import * as Util from "../../utils/util";
-import * as Mixins from "./mixins";
-import { Gettable, FilterableSelectableGettable } from "./actionables";
+import { ITypedHash } from "../../collections/collections";
```

Comment:
Ya, agree - will update. Also good to start a code standards section of wiki with that link as a first thing. Thanks!